keras/layers/core.py

```diff
@@ -644,6 +643,7 @@ def __init__(self, function, output_shape=None,
                  mask=None, arguments=None, **kwargs):
         super(Lambda, self).__init__(**kwargs)
         self.function = function
+        self.input_dtypes = None
```

Comment:
Do you think this should be `self._input_dtypes`? Note that this isnt as well defined as `input_shape`, `input_spec` etc, because a Lambda layer instance could be reused on inputs of different dtypes, and input_dtpyes would simply be the dtype of the last tensor(s) on which the layer was called (which has no significance). So might be cool to differentiate this attr from the `layer.input_*` public API.